types/office-js-preview/index.d.ts

```diff
@@ -69109,6 +69056,12 @@ declare namespace Word {
         */
         toJSON(): Word.Interfaces.ContentControlData;
     }
+    class ContentControlCustom {
+        private getDocument();
+        private getReferenceId();
+    }
+    interface ContentControl extends OfficeExtension.ClientObject, ContentControlCustom {
```

Comment:
Should this interface have a description? Also, I can't make sense of this signature:

interface ContentControl extends OfficeExtension.ClientObject, ContentControlCustom { }

In particular, the comma is puzzling. I've never seen a comma used like this in an `extends` declaration. What does this mean?